### DISCUSSION TITLE
Group table in public schema

### DISCUSSION THREAD

**Discussion by zernonia** [supabase/supabase#7669] [Feature Requests]
Some project will have tons of table, which could be tough to find the table I need. 

Ideally we can use different schema to separate those tables, but there's no way to create a table in other schema in the dashboard. So, my idea is to either allow schema selection when creating table, or group them by the dashboard UI. 

from: 
![image](https://user-images.githubusercontent.com/59365435/178627091-f8a184ae-aed4-4ffc-b430-0f62ba7f8aa4.png)

to: 
![image](https://user-images.githubusercontent.com/59365435/178627782-169d536d-d4d3-40e0-b869-65a36a05a7af.png)

What do you think? 😁

**Comment by Shashankphatkure** [comment]
i actually want this feature from alot of time but still not added!

**Comment by azeunkn0wn** [comment]
This would be nice.

**Comment by MoinJulian** [comment]
This would be useful especially if you have a project with a lot of features where every feature as multiple tables, of course you can prefix tables but that still doesn't make it much clearer. This feature wouldn't even affect the actual postgress database as far as I can tell and would be purely UI based.
